app/options.html

```diff
@@ -0,0 +1,37 @@
+<!--
+ Copyright 2017 NCC Group PLC http://www.nccgroup.trust/
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>NCC Image Checker - Options</title>
+  <link rel="stylesheet" href="styles/options.css">
+  <script src="analytics.js"></script>
+</head>
+<body>
+  <div class="text">
+    <p>NCC Group collects information about your use of this extension, including how many times it is opened and what pages it is used on, using Google Analytics.</p>
```

Comment:
@simonhearne MAINTENANCE - move the description paragraphs inside the options div so it's colocated with the feature it's describing